CHANGES.rst

```diff
@@ -1,6 +1,21 @@
 Release notes
 =============
 
+1.20.0 (2023-09-01)
+-------------------
+
+- feature: Add support for nested fields in ``SPIDERMON_VALIDATION_ERRORS_FIELD`` setting (`PR#416 <https://github.com/scrapinghub/spidermon/pull/417>`_)
+- bug: Fix ItemValidationPipeline using ``__setitem__`` method without ItemAdaper (`PR#415 <https://github.com/scrapinghub/spidermon/pull/415>`_)
+- feature: Improve tests coverage for item validation pipeline (`PR#414 <https://github.com/scrapinghub/spidermon/pull/414>`_) 
+- bug: Fix ``spidermon.utils.zyte.Client`` not following crawler settings (`PR#411 <https://github.com/scrapinghub/spidermon/pull/411>`_)
+- chore: Migrate ReadTheDocs to config file v2 (`PR#408 <https://github.com/scrapinghub/spidermon/pull/408>`_)
+- chore: Refactor package requirements (`PR#407 <https://github.com/scrapinghub/spidermon/pull/407>`_)
+
+Backwards incompatible changes in 1.20.0
+----------------------------------------
```

Comment:
```suggestion
Backward-incompatible changes in 1.20.0
---------------------------------------
```

Or shorter:

```suggestion
Backward-incompatible changes
-----------------------------
```

Reply:
I prefer the first version because the title is at the same indentation level as the 1.20.0 release notes. So, in the future there won't be multiples `Backward-incompatible changes` entries at the same level.

Reply:
I did not notice the level, and I would suggest nesting it to 1.20.0 instead, but no strong opinion.